src/coreComponents/meshUtilities/InternalMeshGenerator.hpp

```diff
@@ -335,11 +300,14 @@ class InternalMeshGenerator : public MeshGeneratorBase
    */
   inline bool isRadial()
   {
-    bool rval = (m_mapToRadial > 0);
+    bool rval = ( m_meshType == MeshType::Cylindrical || m_meshType == MeshType::CylindricalSquareBoundary );
     return rval;
```

Comment:
just `return m_meshType == MeshType::Cylindrical || m_meshType == MeshType::CylindricalSquareBoundary;` does the job.

Reply:
done